apisix/stream/router/ip_port.lua

```diff
@@ -78,9 +78,30 @@ do
 
             local route = item.value
             if route.protocol and route.protocol.superior_id then
-                -- subordinate route won't be matched in the entry
-                -- TODO: check the subordinate relationship in the Admin API
-                goto CONTINUE
+                local admin_api = require("apisix.admin").check_http_admin()
+                if not admin_api then
+                    ngx.log(ngx.ERR, "failed to get http admin node")
```

Comment:
Can we use "core.log" instead?

Reply:
> Can we use "core.log" instead?

@starsz Sorry, I have been out of the station, so replying late. Yeah definitely, I have updated it according to it, please check.